test/migrations/migrator.spec.ts

```diff
@@ -892,9 +893,10 @@ test.group('Migrator', (group) => {
     assert.lengthOf(migrated, 2)
     assert.isTrue(hasUsersTable)
     assert.isTrue(hasAccountsTable)
+    assert.instanceOf(migrator1.error, errors.E_MISSING_SCHEMA_FILES)
     assert.equal(
       migrator1.error!.message,
-      'Cannot perform rollback. Schema file {database/migrations/accounts_v11} is missing'
```

Comment:
Is there any reason the file path was surrounded by `{}`?
That would be a breaking change if another tool uses it to extract the path.